dataframe-openapi/src/test/kotlin/OpenApiTests.kt

```diff
@@ -478,507 +479,609 @@ class OpenApiTests : JupyterReplTestCase() {
-                        convertDataRowsWithOpenApi()
-                        convertTo()
+                
+                      public fun org.jetbrains.kotlinx.dataframe.DataFrame<*>.convertToErrorHolder(convertTo: org.jetbrains.kotlinx.dataframe.api.ConvertSchemaDsl<ErrorHolder>.() -> kotlin.Unit = {}): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = convertTo<ErrorHolder> { 
+                          convertDataRowsWithOpenApi() 
+                          convertTo()
+                      }
+                
+                      public fun readJson(url: java.net.URL): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = org.jetbrains.kotlinx.dataframe.DataFrame
+                          .readJson(url, typeClashTactic = ANY_COLUMNS, keyValuePaths = keyValuePaths)
+                          .convertToErrorHolder()
+                
+                      public fun readJson(path: kotlin.String): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = org.jetbrains.kotlinx.dataframe.DataFrame
+                          .readJson(path, typeClashTactic = ANY_COLUMNS, keyValuePaths = keyValuePaths)
+                          .convertToErrorHolder()
+                
+                      public fun readJson(stream: java.io.InputStream): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = org.jetbrains.kotlinx.dataframe.DataFrame
+                          .readJson(stream, typeClashTactic = ANY_COLUMNS, keyValuePaths = keyValuePaths)
+                          .convertToErrorHolder()
+                
+                      public fun readJsonStr(text: kotlin.String): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = org.jetbrains.kotlinx.dataframe.DataFrame
+                          .readJsonStr(text, typeClashTactic = ANY_COLUMNS, keyValuePaths = keyValuePaths)
+                          .convertToErrorHolder()
                     }
-                    
-                    public fun readJson(url: java.net.URL): org.jetbrains.kotlinx.dataframe.DataFrame<ErrorHolder> = org.jetbrains.kotlinx.dataframe.DataFrame
-                        .readJson(url, typeClashTactic = ANY_COLUMNS, keyValuePaths = keyValuePaths)
-                        .convertToErrorHolder()
+                
+                }
+            }
+            typealias PetRef = `Other advanced test`.Pet
+            typealias AlsoCat = `Other advanced test`.Cat
+            typealias Integer = kotlin.Int
+            typealias SomeArray = org.jetbrains.kotlinx.dataframe.DataFrame<`Other advanced test`.SomeArrayContent>
+            typealias SomeArrayArray = kotlin.collections.List<SomeArray>
+
+            val org.jetbrains.kotlinx.dataframe.ColumnsContainer<`Other advanced test`.Pet>.`value`: org.jetbrains.kotlinx.dataframe.DataColumn<kotlin.Any?> @JvmName("Pet_value") get() = this["value"] as org.jetbrains.kotlinx.dataframe.DataColumn<kotlin.Any?>
```

Comment:
How was this snippet of code was generated (approach/test code)?

Reply:
It just a refactoring of an existing test